plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -307,3 +316,17 @@ def check_rate_limit_error(
                 )
             return 200, None, True
         return status_code, error, False
+
+    def check_for_messages(self, custom_config: Dict[str, Any]) -> None:
+        custom_log_message = custom_config.get(self.CUSTOM_LOG_MESSAGE, None)
```

Comment:
Not necessary to add `None`, but its up to you 😃 

```suggestion
        custom_log_message = custom_config.get(self.CUSTOM_LOG_MESSAGE)
```